src/method_nlar.jl

```diff
@@ -166,17 +99,19 @@ function nlar(::Type{T},
             println("##### Using inner solver:",inner_solver_method," #####");
         end
 
-        while (m < nev) && (k < maxit)
+        while ((m < nev) && (k < maxit))
+            Vk = view(V,:,1:cbs)
             # Construct and solve the small projected PEP projected problem (V^H)T(λ)Vx = 0
             set_projectmatrices!(proj_nep,Vk,Vk);
-
+            
             #Use inner_solve() to solve the smaller projected problem
             dd,vv = inner_solve(inner_solver_method,T,proj_nep,Neig=nev,σ=σ);
 
             # Sort the eigenvalues of the projected problem by measuring the distance from the eigenvalues,
             # in D and exclude all eigenvalues that lie within a unit disk of radius R from one of the
             # eigenvalues in D.
```

Comment:
This comment is too specific, since it already assumes you use a specific sorter.